sdk/personalizer/Azure.AI.Personalizer/src/Models/PersonalizerClient.cs

```diff
@@ -387,7 +388,14 @@ public virtual async Task<Response> RewardAsync(string eventId, float reward, Ca
             try
             {
                 PersonalizerRewardOptions rewardOptions = new PersonalizerRewardOptions(reward);
-                return await EventsRestClient.RewardAsync(eventId, rewardOptions, cancellationToken).ConfigureAwait(false);
+                if (_isLocalInference)
```

Comment:
_isLocalInference only needed for rank, right?

Reply:
We decided to add this for reward and activate api even though adding them are P2.